documentation/src/main/wiki/AccessibilityIssuesReport.md

```diff
@@ -0,0 +1,75 @@
+#### Accessibility Issues Report
+
+This feature allows to generate a summary of various reported accessibility issues for a given suite.
+
+You can access it either through the web GUI, or query the report generation endpoint directly (for integration with build tools like Jenkins, Bamboo, ...).
+
+You can choose between two output file formats:
+- XLSX (Excel) - gives a summary of all issues by given type in a table format
+- TXT (pure text) - targetted as a helper for reporting these issues in JIRA (is already formatted for copy-paste)
```

Comment:
Doesn't it mean that this text is simply formatted using Markdown?

Reply:
JIRA syntax is not exactly Markdown, so stating here that it is MD would be misleading